repair/row_level.cc

```diff
@@ -2665,6 +2668,8 @@ class row_level_repair {
         _zero_rows = false;
         rlogger.debug("ROUND {}, _last_sync_boundary={}, _current_sync_boundary={}, _skipped_sync_boundary={}",
                 master.stats().round_nr, master.last_sync_boundary(), master.current_sync_boundary(), _skipped_sync_boundary);
+        tracing::trace(_trace_state, "negotiate_sync_boundary(): round: {}, _last_sync_boundary: {}, _current_sync_boundary: {}, _skipped_sync_boundary: {}",
```

Comment:
This is going to add massive amount of tracing. I suggest not to add it.

Reply:
We need to have higher resolution, than just the range repair start/finish. Otherwise we will not be able to see, which partition/row the repair gets stuck at. Yes, this will generate a lot of tracing messages, but tracing is off by default, and we can turn it on just for that single range which got stuck or is too slow. This way, we will reduce the overall amount of tracing data.

Reply:
Ok, I'll look into reducing the amount of messages.